### PR TITLE
Add new fields

### PR DESCRIPTION
**Pull request by kelfish**
**Changes**
- Add `standard_webcast` field and convert to `standardWebcast` - `/sessions` endpoint
- Add `youtube_*` fields and convert to `youtube` object - `/sessions` endpoint
- Update `sessions.json` mock data from UAT

### CODE REVIEW

File: src/types/api/virtualVenue.ts [hunk 1]
```diff
@@ -71,6 +71,11 @@ export interface ExternalVenueApiResponse extends ExternalVenueApiResponseList {
         name: string;
       }[];
       premium_webcast: boolean;
+      standard_webcast: boolean;
+      youtube_required: false;
+      youtube_watch_url: string | null;
+      youtube_stream_key: string | null;
+      youtube_ingest_urls: string[];
```
Comment (hollandjake) [inline src/types/api/virtualVenue.ts:78]: I was thinking of grouping all the youtube related ones together, what are your thoughts?

Comment (kelfish) [reply src/types/api/virtualVenue.ts:78]: I did on the conversion, is that what you mean? This are the inbound fields

see https://github.com/BlueElephantDigital/gr-api-js/pull/7/files#diff-51f97b742de30bb162702ea1229aa6314df6664936c65c03496271b07c211264R88

**Comment by hollandjake** [comment]
haha i was in the middle of doing this too lol

File: src/types/session.ts [hunk 2]
```diff
@@ -76,6 +83,10 @@ export interface Session {
   meetingOptions: SessionOption[];
 
   premiumWebcast: boolean;
+  standardWebcast: boolean;
+
+  youtube: YouTube;
```
Comment (kelfish) [inline src/types/session.ts:88]: Instead of this object I could add them as a `watch` role... to keep inline with the others?

Comment (hollandjake) [reply src/types/session.ts:88]: you could, but to be fair i just get rid of the watch rules as fast as possible 🤣

Comment (kelfish) [reply src/types/session.ts:88]: HAHA, ok, will leave like this

**Review by hollandjake — APPROVED** [review]
lgtm

**Pull request merged**